scio-test/src/it/scala/com/spotify/ScioBenchmark.scala

```diff
@@ -377,8 +418,137 @@ private[this] object PrettyPrint {
 // sbt scio-test/it:runMain com.spotify.ScioBatchBenchmarkResult $buildNum1 $buildNum2
 // where $buildNum1 and $buildNum2 are build number of "bench" jobs in CircleCI
 object ScioBatchBenchmarkResult {
+
   import ScioBenchmarkSettings._
+
   def main(args: Array[String]): Unit =
     new DatastoreLogger(BatchMetrics)
       .printMetricsComparison(ScioBatchBenchmark.benchmarkNames, Some((args(0), args(1))))
 }
+
+abstract class Benchmark(val extraConfs: Map[String, Array[String]] = null) extends Serializable {
```

Comment:
can we just make the default value for `extraConfs` an empty `Map` instead of null?